src/vue/forms/TransferForm.spec.js

```diff
@@ -90,50 +96,45 @@ describe('TransferForm component', () => {
     })
     wrapper = shallowMount(TransferForm, {
       store,
+      mixins: [
+        FeesMixin,
+        FormMixin,
+      ],
       localVue,
     })
   })
 
-  describe('loadPaymentFee()', () => {
+  describe('calculateFees()', () => {
```

Comment:
`calculateFees()` is a part of `fees.mixin`. you should not test it within transfer form